alerta/sql/mysql-schema.sql

```diff
@@ -0,0 +1,121 @@
+CREATE TABLE IF NOT EXISTS alerts (
+    id VARCHAR(255),
+    resource VARCHAR(255) NOT NULL,
+    event VARCHAR(255) NOT NULL,
+    environment VARCHAR(255),
+    severity LONGTEXT,
+    correlate json,
+    status LONGTEXT,
+    service json,
+    `group` LONGTEXT,
+    value LONGTEXT,
+    `text` LONGTEXT,
+    tags json,
+    attributes json,
+    origin LONGTEXT,
+    type LONGTEXT,
+    create_time timestamp DEFAULT CURRENT_TIMESTAMP,
+    timeout int,
+    raw_data LONGTEXT,
+    customer VARCHAR(255),
+    duplicate_count int,
+    `repeat` BOOLEAN,
+    previous_severity LONGTEXT,
+    trend_indication LONGTEXT,
+    receive_time timestamp DEFAULT CURRENT_TIMESTAMP,
+    last_receive_id LONGTEXT,
+    last_receive_time timestamp DEFAULT CURRENT_TIMESTAMP,
+    history json,
+    PRIMARY KEY (id)
+);
+
+
+CREATE TABLE IF NOT EXISTS blackouts (
+    id varchar(255),
+    priority TEXT NOT NULL,
+    environment TINYTEXT NOT NULL,
```

Comment:
The schema is something that needs to be review a couple of times. MySQL documentation suggests:

 `Instances of BLOB or TEXT columns in the result of a query that is processed using a temporary table causes the server to use a table on disk rather than in memory because the MEMORY storage engine does not support those data types (see Section 8.4.4, “Internal Temporary Table Use in MySQL”). Use of disk incurs a performance penalty, so include BLOB or TEXT columns in the query result only if they are really needed. For example, avoid using SELECT *, which selects all columns. `

So this needs to be check of how many rows do we need with TEXT(or any of the 4 types of TEXT), and which do we need with varchar(0-65535)